app/elm/Main.elm

```diff
@@ -1,9 +1,13 @@
-module Main exposing (..)
+port module Main exposing (..)
 
 import Html exposing (program)
 import Model exposing (..)
 import Update exposing (..)
 import View exposing (..)
+import Data.Dates exposing (..)
+
+
+port initMap : Marker -> Cmd msg
```

Comment:
I'd suggest moving all the ports into their own module, we can then import the functions from them elsewhere

https://guide.elm-lang.org/interop/javascript.html (near the bottom it suggests only having one port module per app)